src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[NUMBER_OF_BALLS];
+        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
+            balls[i] = new Lottery().getRandomBall();
+            System.out.println(balls[i].toString());
+        }
```

Comment:
```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
            System.out.println(lottery.getRandomBall());
        }
```
![image](https://user-images.githubusercontent.com/81781479/193443229-a2d565d2-fdcc-41b3-8217-1bd71c49f67d.png)

Before you create PR with your homework, you need to go through the checklist under the task and correct all the points described there. The mentor will not check the work until the checklist points are corrected